pkg/remote/builtin/github/github.go

```diff
@@ -34,12 +37,14 @@ type GitHub struct {
 
 func New(conf *config.Config) *GitHub {
```

Comment:
hmm, great question. I was initially thinking of doing something like this so that we could provide default values:

```Go
// create github remote and set required fields
remote := GitHub{}
remote.API = EnvString("GITHUB_HOST", DefaultHost)

// helper function to get environment variable OR default value
func EnvString(name, default) (v string) {
    v = os.Getenv(name)
    if len(v) == 0 {
      v = value
    }
}
```

I'm not opposed to using the `envconfig` library, however.  So whichever method you prefer :)